### ISSUE TITLE
Import Warnings and Freezing

### ISSUE THREAD

**Issue by luischinchillagarcia** [nuxt-community/firebase-module#352]
### Version
@nuxtjs/firebase: 6.1.1
firebase: 8.0.0
nuxt: 2.14.0

### What is actually happening?
Using the most basic default config (shown below), I get a series of warnings:
1. "export 'auth' (imported as 'firebase') was not found in 'firebase/app'    
2. "export 'apps' (imported as 'firebase') was not found in 'firebase/app'  
3. Same warning for the rest of the services... 

Edit: The exact form of the issue is:
```
 WARN  in ./.nuxt/firebase-module/main.js                                                                                     
"export 'functions' (imported as 'firebase') was not found in 'firebase/app'  
```

This also causes my app to completely stop rendering anything. I noticed this because v-model and any event function stopped working. What is the cause of this issue, and is it a configuration issue?

```
 modules: [
    '@nuxt/http',
    '@nuxtjs/axios',
    '@nuxtjs/proxy',
    'nuxt-buefy',
    '@nuxtjs/firebase',
  ],
  firebase: {
    config: {
      apiKey: -
      authDomain: -
      databaseURL: -
      projectId: -
      storageBucket: -
      messagingSenderId: -
      appId: -
      measurementId: -
    },
    services: {
      auth: true,
    }
  },
```

### List of Dependencies

    "@firebase/auth": "^0.15.1",
    "@nuxt/http": "^0.5.13",
    "@nuxtjs/apollo": "^4.0.1-rc.4",
    "@nuxtjs/axios": "^5.12.2",
    "@nuxtjs/firebase": "^6.1.1",
    "@nuxtjs/proxy": "^2.0.1",
    "firebase": "^8.0.0",
    "graphql-tag": "^2.11.0",
    "isomorphic-fetch": "^3.0.0",
    "js-cookie": "^2.2.1",
    "material-icons": "^0.3.1",
    "nuxt": "^2.14.0",
    "nuxt-buefy": "^0.4.3",
    "nuxt-isomorphic-fetch": "^0.1.0",
    "vuesax": "^4.0.1-alpha.25"

**Comment by refoxdev** [comment]
same issue

**Comment by zulfizz** [comment]
i solved this issue with change firebase version to 7.24.0

**Comment by lupas** [comment]
The current version of this module does not support Firebase v8+.

Also see https://github.com/nuxt-community/firebase-module/issues/349, it's already fixed in dev and will be shipped with upcoming v7.0.0 very soon.

**Comment by lupas** [comment]
v7 just released today which will fix this issue. Make sure to carefully read through the [migration guide](https://firebase.nuxtjs.org/community/migrate) when migrate your application. See the release notes here.
